feature/gnoi/system/tests/ping_test/ping_test.go

```diff
@@ -110,7 +111,11 @@ func TestGNOIPing(t *testing.T) {
 	if len(ipv6Addrs) == 0 {
 		t.Fatalf("Failed to get a valid IPv6 loopback address: %+v", ipv6Addrs)
 	}
-
+	if *deviations.ExplicitInterfaceInDefaultVRF {
+		if dut.Vendor() == ondatra.NOKIA {
```

Comment:
I don't think that we need this `dut.Vendor` check here, since the flags can be enabled explictly for specific test runs.

Reply:
That's fair, but the only discrepancy here is that there's an expectation the interface to be part of mgmt NI. If no other vendor require that, I can remove the vendor flag.

Reply:
@Hobbydos, I don't see any part in this test expects the ping from `mgmt` NI. (it's a TODO from the MD though). If so, I think you just need to put the interface in the default VRF, right?